tests/unit/ReferenceListTest.php

```diff
@@ -372,14 +372,4 @@ public function testGivenNonEmptyList_isNotEmpty() {
 		$this->assertFalse( $references->isEmpty() );
 	}
 
-	public function testGivenNonEmptyListWithForwardedIterator_isNotEmpty() {
-		$references = new ReferenceList();
-		$references->addNewReference( new PropertyNoValueSnak( 1 ) );
-		$references->next();
```

Comment:
`IteratorAggregate` does not extend the `Iterator` interface but only `Traversable`. (`IteratorAggregate extends Traversable`) I tested it manually and I cannot call a `next` method on `ReferenceList. (`Call to undefined method Wikibase\DataModel\ReferenceList::next()`)